recipes/r-marquee/meta.yaml

```
@@ -0,0 +1,100 @@
+{% set version = '0.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-marquee
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/marquee_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/marquee/marquee_{{ version }}.tar.gz
+  sha256: ef4b3798d1a85b0898eb03d8a8a4ecc3addebdceefacfb022c8ca37989b51994
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```

Need different stdlib for win:

```suggestion
    - {{ compiler('c') }}        # [not win]
    - {{ stdlib('c') }}          # [not win]
    - {{ compiler('m2w64_c') }}  # [win]
    - {{ stdlib('m2w64_c') }}    # [win]
```